test/specs/apexActions.e2e.ts

```diff
@@ -0,0 +1,211 @@
+    // Get open text editor
+    const workbench = utilities.getWorkbench();
+    await utilities.getTextEditor(workbench, 'SampleDataController.cls');
+
+    // Run 'SFDX: Create Apex Action from This Class'
+    await utilities.executeQuickPick('SFDX: Create Apex Action from This Class', utilities.Duration.seconds(2));
+
+    // Verify 'SFDX: Create Apex Action from This Class' generated the open api file
+    const editorView = workbench.getEditorView();
+    const activeTab = await editorView.getActiveTab();
+    const title = await activeTab?.getTitle();
+    expect(title).to.equal('SampleDataController_openapi.yml');
+
+    // Verify file content
+    const textEditor = await utilities.getTextEditor(workbench, title!);
+    const expectedTexts = [
+      'openapi: 3.0.0',
+      'info:\n\ttitle:',
+      '\tversion',
+      'paths:\n\t',
+      '/importSampleData:\n\t\tpost:',
+      '\t\t\tsummary:',
+      '\t\t\tparameters: []',
+      '\t\t\tresponses'
+    ];
+    verifyFileContent(await textEditor.getText(), expectedTexts);
+  });
+
+  if (process.platform !== 'darwin') {
+    step('SFDX: Create Apex Action from This Class - Editor', async () => {
+      utilities.log(`${testSetup.testSuiteSuffixName} - SFDX: Create Apex Action from This Class - Editor`);
+      // Go to apex class file
+      const workbench = utilities.getWorkbench();
+      const editor = await utilities.getTextEditor(workbench, 'PropertyController.cls');
+      await editor.moveCursor(10, 30);
+      const contextMenu = await editor.openContextMenu();
+      await contextMenu.select('SFDX: Create Apex Action from This Class');
+
+      // Verify 'SFDX: Create Apex Action from This Class' generated the open api file
+      const editorView = workbench.getEditorView();
+      const activeTab = await editorView.getActiveTab();
+      const title = await activeTab?.getTitle();
+      expect(title).to.equal('PropertyController_openapi.yml');
+
+      // Verify file content
+      const textEditor = await utilities.getTextEditor(workbench, title!);
+      const expectedTexts = [
+        'openapi: 3.0.0',
+        'info:\n\ttitle:',
+        '\tversion',
+        'paths:\n\t',
+        '/getPagedPropertyList:\n\t\tpost:',
+        '\t\t\tsummary:',
+        '\t\t\tparameters:',
+        '\t\t\t\t- name: searchKey',
+        '\t\t\t\t\tin: query',
+        '\t\t\t\t\trequired: true',
+        '\t\t\t\t\tdescription:',
+        '\t\t\t\t\tschema:\n\t\t\t\t\t\ttype: string',
+        '\t\t\t\t- name: maxPrice',
+        '\t\t\t\t- name: minBedrooms',
+        '\t\t\t\t- name: minBathrooms',
+        '\t\t\t\t- name: pageSize',
+        '\t\t\t\t- name: pageNumber',
+        '/getPictures:\n\t\tpost:',
+        '\t\t\t\t- name: propertyId',
+        '\t\t\tresponses'
+      ];
+      verifyFileContent(await textEditor.getText(), expectedTexts);
+    });
+  }
+
+  if (process.platform !== 'darwin') {
```

Comment:
Also why is it specific to non-darwin platform?

Reply:
Because mac uses a native element for the context menu, so it doesn't work there. If you check manifest builder, that's the other only place where we have context menu implemented and it's only for windows and ubuntu as well